recipes/bison/all/conanfile.py

```diff
@@ -0,0 +1,164 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from contextlib import contextmanager
+import os
+
+
+SIMPLE_OUTPUT = """\
+#!/usr/bin/env python
```

Comment:
Why don't you save it as an external file?

Reply:
No reason. Just a choice.
exports_sources + os.chmod is the same amount of lines as exports_sources + os.chmod

Reply:
fair enough